util/Halide-VS2017.natvis

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Nice, this is for viewing Halide buffer objects in Visual Studio?

Reply:
Yes, but the visualization may not work on "Release builds with debug information" because `type->bytes()` might get inlined and stripped-out.